src/player.h

```diff
@@ -1310,7 +1310,7 @@ class Player final : public Creature, public Cylinder
 
 	uint32_t getAttackSpeed() const;
 
-	static uint8_t getPercentLevel(uint64_t count, uint64_t nextLevelCount);
+	static uint16_t getBasicPointLevel(uint64_t count, uint64_t nextLevelCount);
```

Comment:
I think it's supposed to be "basi**s** point", not "basi**c** point", it's a typo

Reply:
yeap, typo...